features/support/env.rb

```diff
@@ -40,28 +66,28 @@ def dm_frontend_domain()
   ENV['DM_FRONTEND_DOMAIN']
 end
 
-def dm_admin_uname()
-  ENV['DM_ADMINISTRATORNAME']
+def dm_admin_email()
+  ENV['DM_ADMINISTRATORNAME'] || ENV['DM_ADMIN_EMAIL']
```

Comment:
Are these alternates for old Jenkins vs. new Jenkins?  I'd be tempted to just have the new ones and fix the old Jenkins jobs, but I guess either way is fine (i.e. we can remove the alternates once old Jenkins is turned off) - as long as we remember to tidy up here once old Jenkins is off.